node/overseer/examples/minimal-example.rs

```diff
@@ -169,12 +170,13 @@ fn main() {
 			Delay::new(Duration::from_secs(1)).await;
 		});
 
-		let all_subsystems = AllSubsystems::<()>::dummy()
+		let (overseer, _handle) = dummy_overseer_builder(spawner, AlwaysSupportsParachains, None)
+			.unwrap()
 			.replace_candidate_validation(|_| Subsystem2)
-			.replace_candidate_backing(|orig| orig);
+			.replace_candidate_backing(|orig| orig)
```

Comment:
Is this line not redundant?

Reply:
Yeah, it's a no-op, but it's example code and good to cover.

Reply:
Note that #3868 might be a lot easier to review instead of this.